src/lib/common/include/sol-log.h

```diff
@@ -458,6 +458,24 @@ sol_log_domain_init_level(struct sol_log_domain *domain)
 #define SOL_LOG_LEVEL_POSSIBLE(level) (0)
 #endif
 
+#ifdef SOL_LOG_FILES
+#ifdef SOL_LOG_FUNCTIONS
+#define SOL_LOG_FILE __FILE__
+#define SOL_LOG_FUNCTION __PRETTY_FUNCTION__
+#else
+#define SOL_LOG_FILE __FILE__
+#define SOL_LOG_FUNCTION NULL
+#endif
+#else
+#ifdef SOL_LOG_FUNCTIONS
+#define SOL_LOG_FILE NULL
+#define SOL_LOG_FUNCTION __PRETTY_FUNCTION__
+#else
+#define SOL_LOG_FILE NULL
+#define SOL_LOG_FUNCTION NULL
+#endif
+#endif
```

Comment:
On Fri, 08 Apr 2016 12:06:27 -0700, Gustavo Lima Chaves wrote:
> > +#ifdef SOL_LOG_FUNCTIONS
> > +#define SOL_LOG_FILE __FILE__
> > +#define SOL_LOG_FUNCTION __PRETTY_FUNCTION__
> > +#else
> > +#define SOL_LOG_FILE __FILE__
> > +#define SOL_LOG_FUNCTION NULL
> > +#endif
> > +#else
> > +#ifdef SOL_LOG_FUNCTIONS
> > +#define SOL_LOG_FILE NULL
> > +#define SOL_LOG_FUNCTION __PRETTY_FUNCTION__
> > +#else
> > +#define SOL_LOG_FILE NULL
> > +#define SOL_LOG_FUNCTION NULL
> > +#endif
> > +#endif
> 
> Because go :horse:. I'll fix that too.

And merge it